nilearn/version.py

```diff
@@ -1 +1,97 @@
+# *- encoding: utf-8 -*-
+"""
+nilearn version, required package versions, and utilities for checking
+"""
+# Author: Alexandre Abraham, Philippe Gervais, Ben Cipollini
+# License: simplified BSD
+
 __version__ = '0.1b1'
+
+# All metadata needed for required modules:
+#    * minver: minimum required version of the module
+#    * manual: whether the module must be installed manually
+# See more info in check_module_dependencies docstring.
+#
+# This is a list to preserve order, and to avoid the Python 2.7
+#   dependency of collections.OrderedDict
```

Comment:
This comment should be moved just before the definition of REQUIRED_MODULE_METADATA